Packages/Nakama/Runtime/UnityWebRequestAdapter.cs

```diff
@@ -114,6 +114,45 @@ private UnityWebRequestAdapter()
             Action<ApiResponseException> errback)
         {
             yield return www.SendWebRequest();
+            IsNetworkError(www);

```

Comment:
Could you make `IsNetworkError` a method that returns a bool that only contains the logic that is being changed in this PR?